external-crates/move/crates/move-analyzer/editors/code/src/main.ts

```diff
@@ -132,6 +161,7 @@ export async function activate(extensionContext: Readonly<vscode.ExtensionContex
     context.registerCommand('serverVersion', serverVersion);
     context.registerCommand('build', buildProject);
     context.registerCommand('test', testProject);
+    context.registerCommand('trace', traceProject);
```

Comment:
Does this show up to the user? I'm wondering if it's better to try and hide the fact that there is tracing going on behind the scenes and just expose the concept of "debug" here possibly.

Reply:
Yes, this is what shows up in the command palette. Because of the current structure of the extensions (this is in the move-analyzer extension, the actual debugger is in the trace-debugger directory and another extension), I think this is the best we can do for now. I'm happy to expose this to the user, they're going to be getting used to getting traces and I think the action to think of is more "trace" than "debug."

Reply:
What @tedks said. At this point it's going to be a two-step process:
- generate traces
- start the debugger

We can finesse it for future releases. For example, we can have a drop-down menu by each test to debug this test and this test only: generate traces for this test and immediately launch the debugger. We'd need to tweak test filtering for this as well, though, to have a test execute not when it contains a filter string but only when it matches the filter string exactly